### PR TITLE
cleanup time

### PR DESCRIPTION
**Pull request by asbyth**
(nothing SEEMED broken when i tested, could use testing with beta-testers but afaik nothing i changed broke anything)

- moved config handlers to config/handlers
- remove some unused code
- fix victory royale sound
- remove entitypath variable (HyperiumClient/Hyperium-Repo#108 (accept the hyperium-repo pr after this))
- remove unused json file
- remove unused sounds
- remove unused textures
- add languages Croatian, Hindi, and Chinese
- update copyright (2018 -> 2019)

### CODE REVIEW

File: src/main/java/cc/hyperium/cosmetics/HyperiumCosmetics.java [hunk 1]
```diff
@@ -94,7 +94,6 @@ public HamsterCompanion getHamsterCompanion() {
         DAB,
         BUTT,
         FLIP,
-        DEALWITHIT,
```
Comment (RDIL) [inline src/main/java/cc/hyperium/cosmetics/HyperiumCosmetics.java]: What even...

File: src/main/java/cc/hyperium/mixinsimp/renderer/HyperiumRender.java [hunk 2]
```diff
@@ -6,7 +6,7 @@
 import cc.hyperium.event.RenderNameTagEvent;
 import cc.hyperium.mixins.renderer.IMixinRender;
 import cc.hyperium.utils.ChatColor;
-import cc.hyperium.utils.StaffUtils;
+import cc.hyperium.utils.staff.StaffUtils;
```
Comment (RDIL) [inline src/main/java/cc/hyperium/mixinsimp/renderer/HyperiumRender.java]: Was this a Refactor or mistake?

File: src/main/java/cc/hyperium/config/DefaultConfig.java [hunk 3]
```diff
@@ -18,6 +18,9 @@
 package cc.hyperium.config;
 
 
+import cc.hyperium.config.handlers.PostConfigHandler;
+import cc.hyperium.config.handlers.PreConfigHandler;
```
Comment (RDIL) [inline src/main/java/cc/hyperium/config/DefaultConfig.java]: Whats this about?

File: src/main/java/cc/hyperium/cosmetics/HyperiumCosmetics.java (hunk 1 above)
Comment (asbyth) [reply src/main/java/cc/hyperium/cosmetics/HyperiumCosmetics.java]: wasnt ever a cosmetic

File: src/main/java/cc/hyperium/mixinsimp/renderer/HyperiumRender.java (hunk 2 above)
Comment (asbyth) [reply src/main/java/cc/hyperium/mixinsimp/renderer/HyperiumRender.java]: moved the staffutils class to that package since the other is there aswell

File: src/main/java/cc/hyperium/config/DefaultConfig.java (hunk 3 above)
Comment (asbyth) [reply src/main/java/cc/hyperium/config/DefaultConfig.java]: moved the config handlers to that package

**Comment by RDIL** [comment]
👍 looks good, doesn't seem 🅱️roken for me, except that blue lightning owo (`my texture pack`)

File: src/main/java/cc/hyperium/Hyperium.java [hunk 4]
```diff
@@ -215,27 +241,22 @@ public void init(InitializationEvent event) {
...
-
-            // Register statistics tracking.
-            EventBus.INSTANCE.register(statTrack);
-            CONFIG.register(statTrack);
             CONFIG.register(new ToggleSprintContainer());
+            CONFIG.register(statTrack);
 
             SplashProgress.setProgress(7, I18n.format("splashprogress.startinghyperium"));
-            LOGGER.info("[Hyperium] Started!");
             Display.setTitle("Hyperium " + Metadata.getVersion());
```
Comment (CoalCoding) [inline src/main/java/cc/hyperium/Hyperium.java]: Why has this been removed?

File: src/main/java/cc/hyperium/Hyperium.java [hunk 5]
```diff
@@ -384,11 +406,10 @@ private void shutdown() {
                 e.printStackTrace();
             }
         }
+
         // Tell the modules the game is shutting down
         EventBus.INSTANCE.post(new GameShutDownEvent());
 
-        LOGGER.info("Shutting down Hyperium..");
-
```
Comment (CoalCoding) [inline src/main/java/cc/hyperium/Hyperium.java]: Why has this been removed?

File: src/main/resources/assets/minecraft/sounds.json [hunk 6]
```diff
@@ -2031,47 +2031,11 @@
...
-      }
-    ]
-  },
-  "victory_royale": {
-    "category": "ambient",
-    "sounds": [
-      {
-        "name": "victory-royale",
+        "name": "victoryroyale",
         "stream": false
```
Comment (CoalCoding) [inline src/main/resources/assets/minecraft/sounds.json]: Why was this changed? It's easier to distinguish with the dash.

Comment (asbyth) [reply src/main/resources/assets/minecraft/sounds.json]: wasnt working with the dash for some reason

**Comment by asbyth** [comment]
yo is this ready orrr am i missing something @CoalCoding

File: src/main/java/cc/hyperium/Hyperium.java (hunk 4 above)
Comment (RDIL) [reply src/main/java/cc/hyperium/Hyperium.java]: @asbyth I think he wanted this fixed also

File: src/main/java/cc/hyperium/Hyperium.java [hunk 7]
```diff
@@ -254,6 +254,7 @@ public void init(InitializationEvent event) {
             CONFIG.register(statTrack);
 
             SplashProgress.setProgress(7, I18n.format("splashprogress.startinghyperium"));
+            LOGGER.info("[Hyperium] Started!");
             Display.setTitle("Hyperium " + Metadata.getVersion());
```
Comment (RDIL) [inline src/main/java/cc/hyperium/Hyperium.java]: nvm
